docs-conceptual/azps-11.6.0/authenticate-interactive.md

```diff
@@ -22,55 +22,127 @@ principals or other noninteractive authentication methods.
+  of Windows, Linux, and macOS, Azure PowerShell automatically defaults to a browser.
+- Microsoft Accounts (for example, @outlook.com or @live.com) aren't currently supported. We're
+  working with the Microsoft Identity team to bring this support later.
+
+### Disable WAM
+
+Use the following command to disable WAM for Azure PowerShell and return to browser-based login, the
+default before Az 12.0.0.
+
+```azurepowershell
+Update-AzConfig -EnableLoginByWam $false
+```
 
-1. Sign in with your Azure account credentials in the browser.
+## Browser-based login
 
-If you have access to multiple subscriptions, you're signed in to the first subscription Azure
-returns. Commands run against this subscription by default. To change your active subscription for a
-session, use the `Set-AzContext` cmdlet. To change your active subscription and have it persist
-between sessions on the same system, use the `Select-AzContext` cmdlet.
+Browser-based login is the default interactive login for Linux and macOS systems. Beginning with Az
+PowerShell module version 12.0.0, you must [disable WAM](#disable-wam) for Azure PowerShell to use
+browser-based login on Windows-based systems, which was the default before Az 12.0.0.
 
-> [!IMPORTANT]
-> Your credentials are shared among multiple PowerShell sessions as long as you remain signed in.
-> For more information, see [Azure PowerShell context objects][context-persistence].
+When you sign in interactively with the `Connect-AzAccount` cmdlet, browser-based login opens the
+default web browser to load an Azure sign-in page. Sign in with your Azure account credentials in
+the browser.
 
-### Device code authentication
+## Device code authentication
 
-If no web browser is available or it fails to open, you can force device code flow instead of a
-browser control by specifying the **UseDeviceAuthentication** parameter.
+If Web Account Manager or a web browser is unavailable or it fails to open, you can force device
+code flow by specifying the **UseDeviceAuthentication** parameter.
 
 ```azurepowershell
 Connect-AzAccount -UseDeviceAuthentication
 ```
 
-### Sign in to a different tenant
+## Sign in to a different tenant
 
 If your account is associated with more than one tenant, sign-in requires the **Tenant** parameter
 to be specified when connecting. This parameter works with any sign-in method. When logging in, this
 parameter value can either be the Azure object ID of the tenant (Tenant ID) or the fully qualified
-domain name of the tenant. Due to limitations of the current API, you must use a tenant ID instead
-of a tenant name when connecting with a business-to-business (B2B) account.
+domain name of the tenant.
 
 ```azurepowershell
 Connect-AzAccount -Tenant '00000000-0000-0000-0000-000000000000'
 ```
 
-### Sign in to a national cloud
+Due to limitations of the current API, you must use a tenant ID instead of a tenant name when
```

Comment:
Tenant domain is supported now

Reply:
I've removed the sentence mentioning the limitation.